src/audio/module_adapter/module/iadk_modules.c

```diff
@@ -55,12 +55,15 @@ static int iadk_modules_init(struct processing_module *mod)
 	uint32_t module_entry_point;
 	struct module_data *md = &mod->priv;
```

Comment:
Can you please better describe the problem here. `Latest changes in module adapter` is vague and will confuse the readers in 4 months debugging this code.

Reply:
fixed